toys/LisExpr.chpl

```diff
@@ -349,36 +431,36 @@ module LisExpr
                 return new owned Value(ret);
             }
             when (LVT.Lst) {
-                var lst = ast.toListValue(GenList).lv;
+                ref lst = ast.toListValue(GenList).lv;
                 // no empty lists allowed
                 checkGEqLstSize(lst,1);
                 // currently first list element must be a symbol of operator
-                checkSymbol(lst[1]);
-                var op = lst[1].toListValue(Symbol).lv;
+                checkSymbol(lst[0]);
+                var op = lst[0].toListValue(Symbol).lv;
                 select (op) {
-                    when "+"  {checkEqLstSize(lst,3); return eval(lst[2], env) + eval(lst[3], env);}
-                    when "-"  {checkEqLstSize(lst,3); return eval(lst[2], env) - eval(lst[3], env);}
-                    when "*"  {checkEqLstSize(lst,3); return eval(lst[2], env) * eval(lst[3], env);}
-                    when "<"  {checkEqLstSize(lst,3); return eval(lst[2], env) < eval(lst[3], env);}
-                    when ">"  {checkEqLstSize(lst,3); return eval(lst[2], env) > eval(lst[3], env);}
-                    when "<="  {checkEqLstSize(lst,3); return eval(lst[2], env) <= eval(lst[3], env);}
-                    when ">="  {checkEqLstSize(lst,3); return eval(lst[2], env) >= eval(lst[3], env);}
-                    when "==" {checkEqLstSize(lst,3); return eval(lst[2], env) == eval(lst[3], env);}
-                    when "!=" {checkEqLstSize(lst,3); return eval(lst[2], env) != eval(lst[3], env);}
-                    when "or" {checkEqLstSize(lst,3); return or(eval(lst[2], env), eval(lst[3], env));}
-                    when "and" {checkEqLstSize(lst,3); return and(eval(lst[2], env), eval(lst[3], env));}
-                    when "not" {checkEqLstSize(lst,2); return not(eval(lst[2], env));}
+                    when "+"  {checkEqLstSize(lst,3); return eval(lst[1], env) + eval(lst[2], env);}
```

Comment:
These would need to have some sort of helper, similar to the ones we built for Arkouda, to pull out elements in an index-neutral manner.  Or something like https://github.com/chapel-lang/chapel/issues/15445.